.github/workflows/python_tools.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         python setup.py install
       working-directory: ./tools/validators/ontology_validator
     - name: Ontology Yaml Validator Generator Tests 
-      if: (matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest') 
+      if: (matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest')
```

Comment:
can you please add a section for the instance validator

Reply:
I included the tests in a separate PR, can I add the GitHub workflow section later?

Reply:
Done